src/Illuminate/View/Compilers/Compiler.php

```diff
@@ -26,9 +27,15 @@
      * @param  \Illuminate\Filesystem\Filesystem  $files
      * @param  string  $cachePath
      * @return void
+     *
+     * @throws \InvalidArgumentException
      */
     public function __construct(Filesystem $files, $cachePath)
     {
+        if (! $cachePath) {
+            throw new InvalidArgumentException('The cache path must be non-empty.');
```

Comment:
I meant because it's referring to a path which might be empty (not contain any files).